tsql/tsql.g4

```diff
@@ -230,31 +230,37 @@ create_alter_procedure
 create_alter_function
     : (CREATE | ALTER) FUNCTION func_proc_name
         (('(' procedure_param (',' procedure_param)* ')') | '(' ')') //must have (), but can be empty
-        (function_type1 | function_type2 | function_type3) ';'?
-
-    ;
-
-function_type1:RETURNS TABLE
-               (WITH function_option (',' function_option)*)?
-               AS?
-               RETURN select_statement ;
-
-function_type2: RETURNS LOCAL_ID table_type_definition
-                (WITH function_option (',' function_option)*)?
-                AS?
-                BEGIN
-                   sql_clause*
-                   RETURN
-                END ;
-
-
-function_type3:RETURNS data_type
-               (WITH function_option (',' function_option)*)?
-               AS?
-               BEGIN
-                   sql_clause*
-                   RETURN ret=expression ';'?
-               END ;
+        (funcBody_returns_select | funcBody_returns_table | funcBody_returns_scalar) ';'?
+
+    ;
+
+funcBody_returns_select
```

Comment:
The style of this grammar file is `snake_case`.
So I think `func_body_returns_select` is better than `funcBody_returns_select`.